cmd/oracled/cmd/get_oracle_key.go

```diff
@@ -25,15 +27,22 @@ func getOracleKeyCmd() *cobra.Command {
 				return err
 			}
 
-			svc, err := service.New(conf)
+			sgx := sgx.NewOracleSgx()
+
+			queryClient, err := panacea.LoadVerifiedQueryClient(context.Background(), conf, sgx)
+			if err != nil {
+				return fmt.Errorf("failed to load query client: %w", err)
+			}
+
+			svc, err := service.New(conf, sgx, queryClient)
 			if err != nil {
 				return err
 			}
 
 			ctx := context.Background()
 
-			uniqueID := svc.EnclaveInfo().UniqueIDHex()
-			oracleAddress := svc.OracleAcc().GetAddress()
+			uniqueID := svc.GetEnclaveInfo().UniqueIDHex()
+			oracleAddress := svc.GetOracleAcc().GetAddress()
```

Comment:
```suggestion
			uniqueID := svc.EnclaveInfo().UniqueIDHex()
			oracleAddress := svc.OracleAcc().GetAddress()
```
https://go.dev/doc/effective_go#Getters
We don't need the `Get` prefix in Go. But if you added it for some reason, it's okay to have it.

Reply:
Okay, I'll check this out.
I'm sure there were other reasons, but I think it's safe to put it back now.